test/README.md

```diff
@@ -46,9 +46,67 @@ optional arguments:
   --version Future|vXX   selects the network template file
 ```
 
+Tests in the `e2e_subs/serial` directory are executed serially instead of in parallel. This should only be used when absolutely necessary.
+
+### Running a Single E2E Test
+
 To run a specific test, run e2e.sh with -i interactive flag, and follow the instructions:
+```bash
+$ test/scripts/e2e.sh -i
 ```
-test/scripts/e2e.sh -i
+
+In particular, after 30 seconds or so you'll be prompted with some Python virtual environment related exports. You should open a _new terminal shell_ at that point and do something like:
+
+```bash
+$ export VIRTUAL_ENV="/SOME/VERY/LONG/PATH/TO/ve"
+$ export PATH="$VIRTUAL_ENV/bin:$PATH"
 ```
 
-Tests in the `e2e_subs/serial` directory are executed serially instead of in parallel. This should only be used when absolutely necessary.
+Once these virtual environment var's are in place, you can keep using that shell to run specific E2E tests with a commands such as:
+
+```bash
+python3 test/scripts/e2e_client_runner.py $(pwd)/test/scripts/e2e_subs/your_e2e_script.py
+```
+
+### Interaction through a Remote Debugger in a Python E2E Test Script
```

Comment:
@tsachiherman - Any thoughts?

Reply:
so, I think that adding these readme instructions is great - but it would be even better if we could avoid having the code snippet inside the README and having it implemented into the actual code ( /w an optional argument ).
Then, you'll be able to add a unit test to verify that it's working as designed.